www/js/share/services/alert.service.js

```diff
@@ -196,4 +196,52 @@ angular.module("binary").service("alertService", function($translate, $ionicPopu
             );
         });
     };
+
+    this.showProfessioanlClientInformation = function(scope) {
```

Comment:
@nazaninreihani Since all of the popup messages follow the same structure, it's better to create a factory function and pass the parameters to it instead of repeating them.

Reply:
@morteza-binary done